### PR TITLE
widevine:

### PR DESCRIPTION
**Pull request by arthur-she**
Add widevine_unittest

Signed-off-by: Arthur She <arthur.she@linaro.org>

### CODE REVIEW

File: automated/linux/widevine/widevine_unittest.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+metadata:
+    name: widevine_unittest
+    format: "Lava-Test-Shell Test Definition 1.0"
+    description: "Widevine ce_cdm unit test"
```
Comment (mwasilew) [inline automated/linux/widevine/widevine_unittest.yaml:4]: Please make the description more 'descriptive'. It is displayed in the documentation so oneliners are not a good idea.

**Comment by mwasilew** [comment]
```
Validating file: ./automated/linux/widevine/widevine_unittest.sh
* SHELLCHECK: [FAILED]: ./automated/linux/widevine/widevine_unittest.sh
* SHELLCHECK: [OUTPUT]:
In ./automated/linux/widevine/widevine_unittest.sh line 24:
 if $(grep -q "error while loading shared libraries" "${LOG_FILE}") ; then
    ^-- SC2091: Remove surrounding $() to avoid executing output.

 In ./automated/linux/widevine/widevine_unittest.sh line 26:
     dest_dir=$(dirname ${lib})
                        ^----^ SC2086: Double quote to prevent globbing and word splitting.
 
 In ./automated/linux/widevine/widevine_unittest.sh line 27:
     lib_name=$(basename ${lib})
                         ^----^ SC2086: Double quote to prevent globbing and word splitting.
 
 For more information:
   https://www.shellcheck.net/wiki/SC2091 -- Remove surrounding $() to avoid e...
   https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

**Comment by mwasilew** [comment]
Please rebase on top of latest master. Licence check changed and it no longer accepts 'reuse' command.

**Comment by arthur-she** [comment]
> Please rebase on top of latest master. Licence check changed and it no longer accepts 'reuse' command.

Not sure if my understanding is correct. I've rebased on top of the latest master, however, 'reuse' is still there in .travis.yaml
https://github.com/Linaro/test-definitions/blob/master/.travis.yml#L16

**Comment by mwasilew** [comment]
Sorry, my bad. I fixed it in almost all repositories by now. I'll push the change to master in a moment.

**Comment by mwasilew** [comment]
Master was updated. You can try to rebase again. License check should pass now.

**Comment by arthur-she** [comment]
> Master was updated. You can try to rebase again. License check should pass now.
Done. Thank you so much, Milosz.

**Comment by mwasilew** [comment]
I'm missing 2 things:
 - check whether the widevine CDM unit tests are installed. If they're missing, script should exit gracefully. You're also using gzip without checking if it exists.
 - proper commit message.

**Comment by arthur-she** [comment]
> I'm missing 2 things:
> 
> You're also using gzip without checking if it exists.

Do you mean gzip itself exist or the file to be zipped exist?

**Comment by mwasilew** [comment]
gzip itself

**Comment by arthur-she** [comment]
> gzip itself

Done. If it fails to zip logfile, upload the unzipped file to artifactorial.

**Comment by mwasilew** [comment]
@arthur-she this looks good. Please update commit message and the patch can be merged.

**Comment by arthur-she** [comment]
> @arthur-she this looks good. Please update commit message and the patch can be merged.

Updated.

**Pull request merged**